Sming/compiler/ld/standalone.rom.ld

```diff
@@ -5,7 +5,7 @@ MEMORY
   dport0_0_seg :                        org = 0x3FF00000, len = 0x10
   dram0_0_seg :                         org = 0x3FFE8000, len = 0x14000
   iram1_0_seg :                         org = 0x40100000, len = 0x8000
-  irom0_0_seg :                         org = 0x4020a000, len = (1M - 0x0a000)
```

Comment:
@johndoe8967 Please, remove that change from the PR. It is not related to the issue that you are solving.

Reply:
I think it is, because otherwise I get seg faults, but I removed the change
-> so on MacOS Sming is still not working